### DISCUSSION TITLE
How to create a Button ?

### DISCUSSION THREAD

**Discussion by Kurayami-Senpai** [requarks/wiki#6701] [Help / Questions] [answered]
I'm new to the Wiki.js community and I have a question I would like to create a forward button like it was done on https://docs.requarks.io/en/install to get to the following points. But I didn't find out how to do this instead, even a look at the code didn't help me because I can't insert it. It is always displayed to me as text but never as a button

**Comment by NGPixel** [comment, marked as answer]
You need to use the markdown editor.

See the source for an example:
https://docs.requarks.io/s/en/install

**Reply by Kurayami-Senpai** [reply]
Thanks!
